docker/justfile

```diff
@@ -53,6 +53,13 @@ test: test-js test-py
 
 # run dev server in docker container
 serve env="dev" *args="":
+    #!/usr/bin/env bash
+    if [ "$env" = "prod" ]; then
```

Comment:
```suggestion
    if [ "{{ env }}" = "prod" ]; then
```

Reply:
just command args are not exposed as bash variables, sadly.

Reply:
I mean, it works locally, which suggests that they are

Reply:
Oh! TIL

Reply:
I'm trying this locally, and for me it silently fails, i.e. doesn't do anything. When I include `set -euo pipefail` (see below) it fails like this:
 
```
env: unbound variable
error: Recipe `_create_storage` failed with exit code 1
```

Reply:
🤔 it might be that you were using `$env` from out of your environment somewhere (e.g. in `.env`), instead of the just recipe argument.